grails-app/assets/javascripts/streama/controllers/dash-ctrl.js

```diff
@@ -3,19 +3,6 @@
 angular.module('streama').controller('dashCtrl',
 	function ($scope, apiService, $state, $rootScope, localStorageService, modalService, $stateParams ) {
 
-		if ($rootScope.currentUser.isAdmin) {
-			apiService.settings.list().success(function (data) {
-				var TheMovieDbAPI = _.find(data, {settingsKey: 'TheMovieDB API key'});
-
-				if (!TheMovieDbAPI.value) {
-					alertify.alert('You need to fill out some required base-settings. You will be redirected to the settings page now.', function () {
-						$state.go('settings.settings');
-					});
-				}
-			});
-		}
-
```

Comment:
You cant remove this part of the code, its important for the initial setup, but maybe query against another missing setting, like the upload directory?